7.1/alpine/tools.json

```diff
@@ -565,6 +565,18 @@
       },
       "test": "psecio-parse --version"
     },
+    {
```

Comment:
BTW the tool requires 7.2+, so we omit this or just leave it be for consistency?